### ISSUE TITLE
[ERROR] dagger::client - timeout syncing with peer 6PZkBb9DTk9M9d8Lx1hiJbRwvrSbMTtze8QaW9ja67mt

### ISSUE THREAD

**Issue by rsamban** [GenesysGo/dagger-testnet-issues#6]
**Describe the bug**
Started getting following error after approximately 30 mins. (it happened after last_event_epoch=34) and my node is dropped from the dashboard.

2024-01-03T03:01:20.463398929+00:00 [INFO] dagger_logger::metrics - datapoint: executed_block block_transactions_count=3 block_execution_lapsed_us=3
2024-01-03T03:01:20.463405167+00:00 [INFO] dagger_logger::metrics - datapoint: new_bundlehash new_transaction_count=28441 bundlehash="r1c482XttS9eyZavyYmCYHPpqd6q7g12coMiN6gX1LG"
2024-01-03T03:01:20.942325635+00:00 [INFO] dagger::p2p::graph::dagger - Added node to DAGGER in 0 millis; new size is 800 w/ current nodes 800
2024-01-03T03:01:25.643226450+00:00 [WARN] dagger_logger::metrics - run: submit status: 204
2024-01-03T03:01:31.344676291+00:00 [ERROR] dagger::client - timeout syncing with peer EzNv76ipyDRbLDu3DahzTnEQvVFJaiuv8Z6fWCiNHFqL
2024-01-03T03:01:31.609625028+00:00 [INFO] dagger::p2p::graph::dagger - bundle 2269 finalized!
2024-01-03T03:01:31.609925902+00:00 [INFO] dagger_logger::metrics - datapoint: new_ordered_events new_ordered_events=2
2024-01-03T03:01:31.609970681+00:00 [INFO] dagger_logger::metrics - datapoint: executed_block block_transactions_count=2 block_execution_lapsed_us=5
2024-01-03T03:01:31.609988715+00:00 [INFO] dagger_logger::metrics - datapoint: executed_block block_transactions_count=3 block_execution_lapsed_us=2
2024-01-03T03:01:31.610000892+00:00 [INFO] dagger_logger::metrics - datapoint: new_bundlehash new_transaction_count=28446 bundlehash="6ccxMXB3AcuuiQjtEwpRs5PtqqLMGv1gN4KRi7pToFAi"
2024-01-03T03:01:32.352872928+00:00 [ERROR] dagger::client - timeout syncing with peer CiWAFsUdQ3mWqtDmy1eKF7eUiGVsWHnZaeXQmbAn7BXB
2024-01-03T03:01:36.765591094+00:00 [WARN] dagger_logger::metrics - run: submit status: 204
2024-01-03T03:01:42.010218261+00:00 [ERROR] dagger::client - timeout syncing with peer 6PZkBb9DTk9M9d8Lx1hiJbRwvrSbMTtze8QaW9ja67mt
2024-01-03T03:01:42.356199499+00:00 [ERROR] dagger::client - timeout syncing with peer EzNv76ipyDRbLDu3DahzTnEQvVFJaiuv8Z6fWCiNHFqL
2024-01-03T03:01:52.361718419+00:00 [ERROR] dagger::client - timeout syncing with peer 6PZkBb9DTk9M9d8Lx1hiJbRwvrSbMTtze8QaW9ja67mt
2024-01-03T03:01:52.671896518+00:00 [ERROR] dagger::client - timeout syncing with peer CiWAFsUdQ3mWqtDmy1eKF7eUiGVsWHnZaeXQmbAn7BXB
2024-01-03T03:02:02.364436634+00:00 [ERROR] dagger::client - timeout syncing with peer 6PZkBb9DTk9M9d8Lx1hiJbRwvrSbMTtze8QaW9ja67mt
2024-01-03T03:02:02.674115828+00:00 [ERROR] dagger::client - timeout syncing with peer CTPTcWjQS6YPTSCWAp8YPCnyZFQz3ZrHLENP6qAyb9Md
2024-01-03T03:02:12.394876641+00:00 [ERROR] dagger::client - timeout syncing with peer G8o2eVk6cB6jpipcehdUXsRbiGxytecpkzmMJZe3gxj3
2024-01-03T03:02:12.676623967+00:00 [ERROR] dagger::client - timeout syncing with peer 6PZkBb9DTk9M9d8Lx1hiJbRwvrSbMTtze8QaW9ja67mt
2024-01-03T03:02:22.397859594+00:00 [ERROR] dagger::client - timeout syncing with peer CTPTcWjQS6YPTSCWAp8YPCnyZFQz3ZrHLENP6qAyb9Md
2024-01-03T03:02:22.679648250+00:00 [ERROR] dagger::client - timeout syncing with peer EzNv76ipyDRbLDu3DahzTnEQvVFJaiuv8Z6fWCiNHFqL
2024-01-03T03:02:32.407777176+00:00 [ERROR] dagger::client - timeout syncing with peer CTPTcWjQS6YPTSCWAp8YPCnyZFQz3ZrHLENP6qAyb9Md
2024-01-03T03:02:32.681457338+00:00 [ERROR] dagger::client - timeout syncing with peer 6PZkBb9DTk9M9d8Lx1hiJbRwvrSbMTtze8QaW9ja67mt
2024-01-03T03:02:42.409681824+00:00 [ERROR] dagger::client - timeout syncing with peer CiWAFsUdQ3mWqtDmy1eKF7eUiGVsWHnZaeXQmbAn7BXB
2024-01-03T03:02:42.682996013+00:00 [ERROR] dagger::client - timeout syncing with peer G8o2eVk6cB6jpipcehdUXsRbiGxytecpkzmMJZe3gxj3
2024-01-03T03:02:52.411832298+00:00 [ERROR] dagger::client - timeout syncing with peer EzNv76ipyDRbLDu3DahzTnEQvVFJaiuv8Z6fWCiNHFqL
2024-01-03T03:02:52.687509492+00:00 [ERROR] dagger::client - timeout syncing with peer CiWAFsUdQ3mWqtDmy1eKF7eUiGVsWHnZaeXQmbAn7BXB
2024-01-03T03:03:02.414021763+00:00 [ERROR] dagger::client - timeout syncing with peer G8o2eVk6cB6jpipcehdUXsRbiGxytecpkzmMJZe3gxj3
2024-01-03T03:03:02.689170348+00:00 [ERROR] dagger::client - timeout syncing with peer EzNv76ipyDRbLDu3DahzTnEQvVFJaiuv8Z6fWCiNHFqL
2024-01-03T03:03:12.416442523+00:00 [ERROR] dagger::client - timeout syncing with peer CTPTcWjQS6YPTSCWAp8YPCnyZFQz3ZrHLENP6qAyb9Md
2024-01-03T03:03:12.690867135+00:00 [ERROR] dagger::client - timeout syncing with peer 6PZkBb9DTk9M9d8Lx1hiJbRwvrSbMTtze8QaW9ja67mt
2024-01-03T03:03:22.418663239+00:00 [ERROR] dagger::client - timeout syncing with peer EzNv76ipyDRbLDu3DahzTnEQvVFJaiuv8Z6fWCiNHFqL
2024-01-03T03:03:22.695717148+00:00 [ERROR] dagger::client - timeout syncing with peer G8o2eVk6cB6jpipcehdUXsRbiGxytecpkzmMJZe3gxj3
2024-01-03T03:03:32.423754767+00:00 [ERROR] dagger::client - timeout syncing with peer 6PZkBb9DTk9M9d8Lx1hiJbRwvrSbMTtze8QaW9ja67mt
2024-01-03T03:03:32.702076700+00:00 [ERROR] dagger::client - timeout syncing with peer CiWAFsUdQ3mWqtDmy1eKF7eUiGVsWHnZaeXQmbAn7BXB
2024-01-03T03:03:42.426127229+00:00 [ERROR] dagger::client - timeout syncing with peer CTPTcWjQS6YPTSCWAp8YPCnyZFQz3ZrHLENP6qAyb9Md
2024-01-03T03:03:42.707572587+00:00 [ERROR] dagger::client - timeout syncing with peer EzNv76ipyDRbLDu3DahzTnEQvVFJaiuv8Z6fWCiNHFqL
2024-01-03T03:03:52.427763474+00:00 [ERROR] dagger::client - timeout syncing with peer CTPTcWjQS6YPTSCWAp8YPCnyZFQz3ZrHLENP6qAyb9Md
2024-01-03T03:03:52.709007316+00:00 [ERROR] dagger::client - timeout syncing with peer 6PZkBb9DTk9M9d8Lx1hiJbRwvrSbMTtze8QaW9ja67mt

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by tracy-codes** [comment]
Can you please attach the entire log file?

**Comment by rsamban** [comment]
[config.log.tgz](https://github.com/GenesysGo/dagger-testnet-issues/files/13820176/config.log.tgz)
This file has following files. Each of them are copies of config.log files when I had that issue. 
config.log_20240102
config.log_20240102_1
config.log_20240102_2
config.log_20240103
config.log


My node was showing up in the dashboard yesterday for some time, before I started having this issue

Let me know

Thank you

**Comment by rsamban** [comment]
Following is my analysis of file config.log_20240103

1. I started my node on 2024-01-03T04:55:39.033248188+00:00
2. It was running good till "2024-01-03T05:15:57.119904579+6. I see some " could not digest full peer sync: start 3 vs 3; graph events 0 vs 0" right before the timout errors" 00:00 [INFO] dagger::p2p::graph::dagger - Cluster Peers:"
3. First timeout "2024-01-03T05:17:08.721896434+00:00 [ERROR] dagger::client - timeout syncing with peer 6PZkBb9DTk9M9d8Lx1hiJbRwvrSbMTtze8QaW9ja67mt"
4. Last "finalized" entry in the log on "2024-01-03T05:17:01.773934665+00:00 [INFO] dagger::p2p::graph::dagger - bundle 7650 finalized!"
5. After that only timeouts and errors for ~ 7 hours (till 2024-01-03T13:39:14.310569826+00:00) at which point I had to restart my node.
6. After timout errors, there is no "finalized" messages or the table that lists all the pubkey/stake_weight/last_event_epoch/hw_info info.

My node was showing up briefly yesterday (and before then), so I believe my set up is right
